Taarafo.Core.Tests.Unit/Services/Foundations/Profiles/ProfileServiceTests.Logic.Modify.cs

```diff
@@ -0,0 +1,79 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Taarafo.Core.Models.Profiles;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Profiles
+{
+    public partial class ProfileServiceTests
+    {
+        [Fact]
+        public async Task ShouldModifyProfileAsync()
+        {
+            // given
+            DateTimeOffset randomDate =
```

Comment:
My thought - break the line only for 120 and belly patterns.

Reply:
done